app/navigation/routes.js

```diff
@@ -167,6 +177,14 @@ export const Routes = {
         navigationProps: {
             title: {id: 'mobile.routes.user_profile', defaultMessage: 'Profile'}
         }
+    },
+    Saml: {
+        key: 'Saml',
+        transition: RouteTransitions.Horizontal,
+        component: Saml,
+        navigationProps: {
+            title: {id: 'mobile.routes.saml', defaultMessage: 'Single SignOn'}
```

Comment:
Is there a server setting to change what SAML is called? We might need to figure out how to customize this without recompiling the app